x-pack/legacy/plugins/epm/server/lib/elasticsearch/template/template.test.ts

```diff
@@ -5,11 +5,22 @@
  */
 
 import { getTemplate, generateMappings } from './template';
-import { readFileSync, writeFileSync } from 'fs';
+import { readFileSync } from 'fs';
 import path from 'path';
 import { safeLoad } from 'js-yaml';
 import { Field, processFields } from '../../fields/field';
 
+// Add our own serialiser to just do JSON.stringify
+expect.addSnapshotSerializer({
```

Comment:
@jfsiii This code is currently duplicated. What is the best place to share testing code? Should this also happen in a test file for example in `lib` and then be imported?